cmd/medius/images/push.go

```diff
@@ -107,48 +108,89 @@ func NewPublishImagesCommand(options *common.Options) *cobra.Command {
 }
 
 func (b *buildAndPublish) Do(entry *common.Entry, timestamp time.Time) ([]string, error) {
-	metadata := entry.Artifact.Metadata()
-	artifactInfo, err := entry.Artifact.Inspect()
-	if err != nil {
-		return nil, fmt.Errorf("error introspecting artifact %q: %v", metadata.Describe(), err)
-	}
-	b.Log.Infof("Remote artifact checksum: %q", artifactInfo.SHA256Sum)
+	var containerDisks []v1.Image
+	var files []string
 
-	imageSha, err := b.getImageSha(metadata.Describe())
-	if err != nil {
-		return nil, err
-	}
-	if imageSha == artifactInfo.SHA256Sum && !b.Options.PublishImagesOptions.ForceBuild {
-		b.Log.Info("Nothing to do.")
-		return nil, nil
-	}
-	if errors.Is(b.Ctx.Err(), context.Canceled) {
-		return nil, b.Ctx.Err()
-	}
+	for i := range entry.Artifacts {
+		metadata := entry.Artifacts[i].Metadata()
+		artifactInfo, err := entry.Artifacts[i].Inspect()
+		if err != nil {
+			b.Log.Errorf("error introspecting artifact %q: %v", metadata.Describe(), err)
+			continue
```

Comment:
The build should fail if it fails to build the disk. It should not silently continue.

Reply:
Okay, will modify it.